src/patternfly/components/Wizard/examples/Wizard.md

```diff
@@ -12,10 +12,12 @@ import './Wizard.css'
 ```hbs isFullscreen
 {{#> wizard wizard--id="wizard-basic"}}
   {{#> wizard-header}}
-    {{#> button button--modifier="pf-m-plain pf-c-wizard__close" button--attribute='aria-label="Close"'}}
-      <i class="fas fa-times" aria-hidden="true"></i>
-    {{/button}}
-    {{#> title title--modifier="pf-m-3xl pf-c-wizard__title"}}Wizard title{{/title}}
+    {{> wizard-close}}
+    {{#> wizard-title}}
```

Comment:
When I see a repeated, standard blocks like this (esp `component__title-text`), I try to add a `.hbs` param to the `title` partial. So, instead of needing to repeat the `wizard-title-text` block, you could just say `{{> wizard-title wizard-title-text--value="My title"}}` and save a couple lines of code for each example. If it needs to be updated later, all the html config lives in one place instead of 5 or 100 or however many times it's used. 

So in the `title` partial:

```
<div class="pf-c-wizard__title{{#if wizard-title--modifier}} {{wizard-title--modifier}}{{/if}}"
  {{#if wizard-title--attribute}}
    {{{wizard-title--attribute}}}
  {{/if}}>
  {{#if wizard-title-text--value}}
    {{#> wizard-title-text}}
      {{{wizard-title-text}}}
    {{/wizard-title-text}}
  {{/if}}
  {{> @partial-block}}
</div>
```